docs/end2end-TrecQA-eval.md

```diff
@@ -22,8 +22,7 @@ Possible parameters are:
 -input (required)
 ```
 
-Path of a TrecQA file
-
+Path of the TrecQA test file. It can be downloaded from [here](http://cs.jhu.edu/~xuchen/packages/jacana-qa-naacl2013-data-results.tar.bz2)
```

Comment:
Right, you need them all at different times. The note there says "Path of **the** TrecQA **test** file", so perhaps "Path to the {dev,train,test} TrecQA file (from here) depending".